src/cosmosdb-preview/azext_cosmosdb_preview/custom.py

```diff
@@ -555,16 +555,21 @@ def cli_cosmosdb_managed_cassandra_datacenter_create(client,
                                                      data_center_location,
                                                      delegated_subnet_id,
                                                      node_count,
-                                                     base64_encoded_cassandra_yaml_fragment=None):
+                                                     base64_encoded_cassandra_yaml_fragment=None,
+                                                     sku=None,
+                                                     disk_sku=None,
+                                                     disk_capacity=None):
 
     """Creates an Azure Managed Cassandra Datacenter"""
 
     data_center_properties = DataCenterResourceProperties(
         data_center_location=data_center_location,
         delegated_subnet_id=delegated_subnet_id,
         node_count=node_count,
-        base64_encoded_cassandra_yaml_fragment=base64_encoded_cassandra_yaml_fragment
-    )
+        base64_encoded_cassandra_yaml_fragment=base64_encoded_cassandra_yaml_fragment,
```

Comment:
What about update, can a customer change sku for new nodes when they are scaling or is it set only once ?